### ISSUE TITLE
Docker Container

### ISSUE THREAD

**Issue by recover-sanity** [arcan1s/ahriman#50]
## Feature summary

Create a Docker container

### Cause of the feature request

Unable to install and use ahriman after multiple attempts while following documentation would like to just use a Docker container that just works™

### Proposed changes and/or features

Create a Docker container so for ease of installation

**Comment by arcan1s** [comment]
Hey there! 

I've added docker image which can be found at https://hub.docker.com/r/arcan1s/ahriman. Currently there is only `latest` image which is built against current master (thus the full name is `arcan1s/ahriman:latest`), but later I will create also tags for the specific package versions.

I've tested basic functionality like web interface, package building and updating process with vanilla configuration (no reports, no sync, no auth), but other features should work more or less.

More detailed description (including available environment variables) can be found at [faq](https://github.com/arcan1s/ahriman/blob/master/docs/faq.md#docker-image).
